### DISCUSSION TITLE
I connected GoCHain mainnet with metamask but sent ETH has gone to ETH mainnet

### DISCUSSION THREAD

**Discussion by focusedbrain** [gochain-io/docs#108] [General]
Hello,

I would like to use Gochain. The issue is that I send money to a metamask address within GoChain mainnet but it has been received by the ETH mainnet. Like this, I am not able to deploy the contract to the gochain mainnet.

**Comment by treeder** [comment]
ETH is on Ethereum so you can only send it to Ethereum addresses.
